lang/build.gradle

```diff
@@ -54,6 +56,12 @@ dependencies {
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.0'
     testImplementation 'org.junit.vintage:junit-vintage-engine:5.7.0'
     testImplementation 'org.junit.jupiter:junit-jupiter-params:5.7.0'
+
+    jmh 'org.openjdk.jmh:jmh-core:0.9'
+    jmh 'org.openjdk.jmh:jmh-generator-annprocess:0.9'
+
+    implementation 'org.openjdk.jmh:jmh-core:0.9'
+    implementation 'org.openjdk.jmh:jmh-generator-annprocess:0.9'
```

Comment:
You're adding a dependency on JMH to the maven package, here.  We don't want that.  I think the reason this is required is because the benchmarks reside in under the `src` directory.